crates/anvil/core/src/eth/transaction/alloy.rs

```diff
@@ -824,9 +827,12 @@ impl TypedTransaction {
             TypedTransaction::Legacy(tx) => *tx.signature(),
             TypedTransaction::EIP2930(tx) => *tx.signature(),
             TypedTransaction::EIP1559(tx) => *tx.signature(),
-            TypedTransaction::Deposit(_) => {
-                Signature::from_scalars_and_parity(B256::ZERO, B256::ZERO, false).unwrap()
-            }
+            TypedTransaction::Deposit(_) => Signature::from_scalars_and_parity(
+                B256::with_last_byte(1),
+                B256::with_last_byte(1),
+                false,
```

Comment:
why is this 1 vs before it was 0?

Reply:
See new description—it's not possible to construct an `alloy_primitives::Signature` with zero fields, so I chose another number arbitrarily. cc @mattsse if there's a consideration i'm missing happy to change it as needed

Reply:
let's add a `Signature::empty()` helper or osmething then?

Reply:
this looks like a hack, we don't want to support it everywhere